DataConnectors/SalesforceServiceCloud/SalesforceSentinelConnector/function.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scriptFile": "__init__.py",
+    "bindings": [
+      {
+        "name": "mytimer",
+        "type": "timerTrigger",
+        "direction": "in",
+        "schedule": "0 0 * * * *"
```

Comment:
No scheduled trigger, is it intended?

Reply:
"schedule": "0 0 * * * *" - Scheduled to run every 1 hour (at 00min:00sec).